apps/dash-lunar-lander/app.py

```diff
@@ -0,0 +1,435 @@
+            marks={0.5: {"label": "Gravity"}},
+        ),
+        html.Div(
+            [
+                html.H5(
+                    "Parameters", style={"width": "25%", "display": "inline-block"}
+                ),
+                html.H5("Outputs", style={"width": "25%", "display": "inline-block"}),
+                html.H5("Adjust Inital Position", style={"display": "inline-block"}),
+            ]
+        ),
+        html.Div(
+            [
+                # Display Current Spacecraft Parameters
+                html.Div(
+                    [
+                        html.P("Inital Mass (kg):", id="m0Out"),
+                        html.P("Max Thrust (N):", id="maThrustOut"),
+                        html.P("ISP (s):", id="ispOut"),
+                        html.P("C3 (rad/s):", id="c3Out"),
+                        html.P("Gravity (N):", id="gravOut"),
+                    ],
+                    style={"width": "24%", "display": "inline-block"},
+                ),
+                html.Div(
+                    [
+                        # Choose Between Different Cost Functions
+                        dcc.RadioItems(
+                            options=[
+                                {"label": "Mass Optimal Control", "value": 1},
+                                {"label": "Time Optimal Control", "value": 2},
+                            ],
+                            id="costFun",
+                            value=1,
+                        ),
+                        # Display Final Cost Functions
+                        html.P("Final  Mass (kg):", id="mfOut"),
+                        html.P("TOF (s):", id="tof"),
+                    ],
+                    style={"width": "24%", "display": "inline-block"},
+                ),
+                # DPad for Adjusting the Spacecrafts Initial Position
+                html.Div(
+                    [html.Button("Left", id="Left",),],
+                    style={
+                        "width": "15.84%",
+                        "height": "100%",
+                        "display": "inline-block",
+                    },
+                    className="direction-button",
+                ),
+                html.Div(
+                    [html.Button("Up", id="Up",), html.Button("Down", id="Down",),],
+                    style={"width": "15.84%", "display": "inline-block"},
+                    className="direction-button",
+                ),
+                html.Div(
+                    [
+                        html.Button(
+                            "Right",
+                            id="Right",
+                            style={
+                                "width": "100%",
+                                "height": "100%",
+                                "vertical-align": "middle",
+                            },
+                        ),
+                    ],
+                    style={"width": "15.84%", "display": "inline-block"},
```

Comment:
![Screen Shot 2020-01-27 at 1 02 42 PM](https://user-images.githubusercontent.com/30911511/73201473-14565480-4107-11ea-883b-4f11785cb8fd.png)

Is it possible to put separate containers around these three sections? I'm not sure how nice this app is supposed to look, but it looks a little messy. For example, I'm not a fan of the awkward spacing between "Outputs" and the radio buttons.

Reply:
this was a bit of work but it's updated in the latest commit 💯